src/commands/create/groups/quickstart.ts

```diff
@@ -99,17 +162,17 @@ export default async (folderLocation: string, folderRelativePath: string, templa
     ...env,
     WALLET_PRIVATE_KEY: privateKey,
   };
-  const packageManager = await askForPackageManager();
+
+  const packageManager: PackageManagerType = isFramework(template, "Foundry") ? "forge" : await askForPackageManager();
```

Comment:
This part is kind of confusing to me. If user selected Foundry template then I guess we shouldn't ask him what package manager to use, since `forge` is the only option, right? Lmk if I'm wrong, cause I don't have too much experience with foundry

And then in `logFoundryInfo` you shouldn't use `packageManagers[packageManager]` since it makes it less clear which package manager you're actually using.

Reply:
@JackHamer09 that is correct, however they can still select the hardhat quickstart templates of which they would proceed with the ` askForPackageManager();` flow.

Reply:
For `logFoundryInfo` that I will update. However with I think its still necessary to use this conditional check in case user makes use of hardhat quickstart templates.